server/spec/content_delivery_network_spec.rb

```diff
@@ -70,4 +70,84 @@
     cdn.certificate['cert'].should == "test-cert"
   end
 
+  it 'should allow candlepin to create certificate serial' do
```

Comment:
The API currently allows this and I just added a test for it since it was missing. If the serial is included with the certificate provided in the request body when creating the CDN, candlepin will persist it. If we allow them to set it manually when creating the certificate, why not let them be able to update it?